packages/react-components/src/components/RTE/RichTextEditor.tsx

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import * as React from 'react';
```

Comment:
Could you update it to `import React from 'react';` to be consistent with the other code?

Reply:
It looks like  the conversation is resolved by mistake as the code wasn't updated